tests/json/schema_1_orient/input.py

```diff
@@ -0,0 +1,62 @@
+import numpy as np
```

Comment:
regarding test names, any reason `schema_1_orient` can't go to `schema-1-orient`? I've gotten the whole test suite standardized on dashes, no underscores.

Reply:
yes, please.